script/model.py

```diff
@@ -4,141 +4,350 @@
 import numpy as np
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
-import torch.optim as optim
-from torch.utils.data import Dataset, DataLoader
-
-PI = torch.tensor(np.pi)
-
-class MNIST_Dataset(Dataset):
-    def __init__(self, image):
-        super(MNIST_Dataset).__init__()
-        self.image = image
-    def __len__(self):
-        return self.image.shape[0]
-    def __getitem__(self, idx):
-        image = np.copy(self.image[idx, :].reshape(28,28))
-        image = image.astype(np.float32)
-        direction = np.random.choice(['left', 'right'])
-        image_list = []
-        image_list.append(image.reshape(-1))
-        for k in range(1,20):
-            if direction == 'left':
-                tmp = np.delete(image, 0, 1)
-                image = np.concatenate((tmp, image[:,0].reshape(-1,1)) ,1)
-                image_list.append(image.reshape(-1))
-            elif direction == 'right':
-                tmp = np.delete(image, -1, 1)
-                image = np.concatenate((image[:,-1].reshape(-1,1), tmp) ,1)
-                image_list.append(image.reshape(-1))
-                
-        image_seq = np.array(image_list)
-        return image_seq
+
+
+# nn.Module is a class
+class DBlock(nn.Module):
+    """ A basic building block for parametralize a normal distribution.
+    It is corresponding to the D operation in the reference Appendix.
+
+    we denote D the map from x to a normal distribution with mean μ(x) and 
+    log-standard deviation log σ(x)
+
+    We use the letter D for all such maps (even when they don’t 
+    share weights)
+
+    reparameterization trick: randomly sample ε from a unit Gaussian, 
+    and then shift the randomly sampled ε by the latent distribution's mean 
+    μ and scale it by the latent distribution's variance σ.
+    """
+
+    def __init__(self, input_size, hidden_size, output_size):
+        super(DBlock, self).__init__()
+        self.input_size = input_size
+        self.hidden_size = hidden_size
+        self.output_size = output_size
+
+        # fc = fully connected layer 
```

Comment:
```suggestion
        # fc means "fully connected layer" 
```

To avoid confusing a reader that you're not actually commenting out the setting of a variable